mod.d.ts

```diff
@@ -0,0 +1,117 @@
+export declare const KEY_CANCEL = 3;

```

Comment:
I think we need to `declare module` somewhere because we are writing this for a JS build?